client/src/components/Auth/Signup.js

```diff
@@ -1,5 +1,60 @@
-const Signup = () => {
-  return <div className="signup-wrapper"></div>;
+const Signup = (props) => {
+  const state = {
+    username: '',
+    password: '',
+  };
+
+  const handleChange = (e) => {
+    state[e.target.name] = e.target.value;
+  };
+
+  return (
```

Comment:
I have done the layout implementation for this so I will have to kind of utilize your implementation here to integrate it with my front end UI.